src/main/java/nextstep/favorite/domain/Favorite.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.favorite.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import nextstep.subway.domain.BaseEntity;
+
+@Entity
+public class Favorite extends BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long source;
+    private Long target;
+
+    public Favorite() {
+    }
+
+    public Favorite(Long source, Long target) {
```

Comment:
```suggestion
    public Favorite(Station source, Station target) {
```
long 타입으로 받게 되면 아이디가 아닌 어떤 long 타입 값이든 전달이 가능할테니 역 객체를 받아와서 직접 아이디만 추출해주는 방식으로 하는걸 추천합니다.

Reply:
아.. 즐겨찾기 객체 안정성 자체가 좋아지는 느낌이네요 ㅎㅎ 수정해보았습니다